server/src/main/kotlin/ir/armor/tachidesk/model/database/table/ChapterTable.kt

```diff
@@ -26,18 +26,18 @@ object ChapterTable : IntIdTable() {
     val chapterIndex = integer("index")
 
     val manga = reference("manga", MangaTable)
-}
 
-fun ChapterTable.toDataClass(chapterEntry: ResultRow) =
-    ChapterDataClass(
-        chapterEntry[ChapterTable.url],
-        chapterEntry[ChapterTable.name],
-        chapterEntry[ChapterTable.date_upload],
-        chapterEntry[ChapterTable.chapter_number],
-        chapterEntry[ChapterTable.scanlator],
-        chapterEntry[ChapterTable.manga].value,
-        chapterEntry[ChapterTable.isRead],
-        chapterEntry[ChapterTable.isBookmarked],
-        chapterEntry[ChapterTable.lastPageRead],
-        chapterEntry[ChapterTable.chapterIndex],
-    )
```

Comment:
doing it this way is also a convention, having a table only have column definitions is more natural and also a Exposed convention to me

Reply:
My main annoyence of it was the redundant qualifier name that always happened with these extension functions, I tried another way of fixing it in the last commit